source/TLS.rst

```diff
@@ -28,102 +28,295 @@ Operator yourself, as well as how to temporarily disable it if needed.
+
+   .. code:: bash
+
+      $ cat ca.pem ca.pem.old >> ca.pem.combined
+ 
+#. Create a new Secrets object with *old* TLS certificate (``tls.pem.old``)
+   and key (``tls.key.old``), but a *new combined* ``ca.pem``
+   (``ca.pem.combined``):
+
+   .. code:: bash
+
+      $ kubectl delete secret/my-cluster-ssl-internal
+      $ kubectl create secret generic my-cluster-ssl-internal --from-file=tls.crt=tls.pem.old --from-file=tls.key=tls.key.old --from-file=ca.crt=ca.pem.combined --type=kubernetes.io/tls
+
+#. The cluster will go through a rolling reconciliation, but it will do it
+   without problems, as every node has old TLS certificate/key, and both new
+   and old CA certificates.
+
+#. If new TLS certificate and key weren't generated on step 1,
+   :ref:`do that <tls.certs.manual>` now.
+
+#. Create a new Secrets object for the second time: use new TLS certificate
+   (``server.pem`` in the example) and its key (``server-key.pem``), and again
+   the combined CA certificate (``ca.pem.combined``):
+
+   .. code:: bash
+
+      $ kubectl delete secret/my-cluster-ssl-internal
+      $ kubectl create secret generic my-cluster-ssl-internal --from-file=tls.crt=server.pem --from-file=tls.key=server-key.pem --from-file=ca.crt=ca.pem.combined --type=kubernetes.io/tls
+
+#. The cluster will go through a rolling reconciliation, but it will do it
+   without problems, as every node already has a new CA certificate (as a part
+   of the combined CA certificate), and can successfully allow joiners with new
+   TLS certificate to join. Joiner node also has a combined CA certificate, so
+   it can authenticate against older TLS certificate.
+
+#. Create a final Secrets object: use new TLS certificate (``server.pmm``) and
+   its key (``server-key.pem``), and just the new CA certificate (``ca.pem``):
+
+   .. code:: bash
+
+   $ kubectl delete secret/my-cluster-ssl-internal
+   $ kubectl create secret generic my-cluster-ssl-internal --from-file=tls.crt=server.pem --from-file=tls.key=server-key.pem --from-file=ca.crt=ca.pem --type=kubernetes.io/tls
+
+#. The cluster will go through a rolling reconciliation, but it will do it
+   without problems: the old CA certificate is removed, and every node is
+   already using new TLS certificate and no nodes rely on the old CA
+   certificate any more.
+
+.. _tls.cets.update.with.downtime:
+
+Update certificates with downtime
+---------------------------------
+
+If your certificates have been already expired (or if you continue to use the
+Operator version prior to 1.9.0), you should move through the
+*pause - update Secrets - unpause* route as follows.
+
+#. Pause the cluster :ref:`in a standard way<operator-pause>`, and make
+   sure it has reached its paused state.
+
+#. Delete Secrets to force the SSL reconciliation:
+
+   .. code:: bash
+
+      $ {
+        kubectl delete issuer/cluster1-pxc-ca
+        kubectl delete certificate/cluster1-ssl certificate/cluster1-ssl-internal
```

Comment:
only one thing. These lines are valid only in case cert-managet is used. I think we need to move it to a separate point with additional description (if cert-manager is used)